react/store-global.css

```diff
@@ -1,5 +1,27 @@
 @import url(//io.vtex.com.br/fonts/fabriga/stylesheet.css);
 
+/*
+  Media Query Extensions:
+  -s  = 20em
+  -ns  = 40em
+  -m  = 40em
+  -l  = 64em
+  -xl  = 80em
+*/
+:root {
+  --font-family: Fabriga, -apple-system, BlinkMacSystemFont, avenir next, avenir,
```

Comment:
Why we need fabriga font? 🤔

Reply:
It was there before, and i thought that was necessary. Don't it ?